pandas/core/common.py

```diff
@@ -1596,7 +1596,7 @@ def is_timedelta64_dtype(arr_or_dtype):
 
 
 def is_timedelta64_ns_dtype(arr_or_dtype):
-    tipo = _get_dtype_type(arr_or_dtype)
+    tipo = _get_dtype(arr_or_dtype)
```

Comment:
no, to be consistent this should not change (as it will then be inconsisten with everything else). what is causing the issue?